src/main/java/generator/BaseScenarioGenerator.java

```diff
@@ -62,10 +62,62 @@ public Model generate() throws NoSuchMethodException, IllegalAccessException, In
         addEnvData(model);
         addSurrounding(model);
         addMainVehicle(model);
-
+        addPedestrian(model);
+//        addVehicle(model);
         return model;
     }
 
+    private void addVehicle(Model model) {
```

Comment:
jak jest z tym BasicScenarioGenerator, to jest w 100% nasze, poskładane z fragmentów innych generatorów?

Reply:
BaseScenatrioGenerator to jest ich generator + dodałem te 2 metody addVehicle i addPedestrian żeby sobie łatwo testować + jeszcze dodałem valueInDollars